### PR TITLE
Auth0 integration

### LINKED ISSUE #395: Add Auth0 Integration
We need to complete Auth0 integration at Atheneum. 

We must also add an Auth0 badge at the site - see our README -, in order to be elegible to OSS plan.

### PR DESCRIPTION
**Pull request by flbulgarelli**
Fixes #395

### CODE REVIEW

**Comment by flbulgarelli** [comment]
screencapture-central-localmumuki-io-3000-1452269905232

File: README.md [hunk 1]
```diff
@@ -9,10 +9,19 @@ Mumuki Atheneum
...
+## About
 
 Atheneum is a multitenant Rails webapp for solving exercises, organized in terms of chapters and guides.
 
+## Running
+
+```bash
+MUMUKI_AUTH0_CLIENT_ID=...\
+MUMUKI_AUTH0_CLIENT_SECRET=... \
+MUMUKI_AUTH0_DOMAIN=... \
```
Comment (faloi) [inline README.md]: I hate you :angry:

File: app/models/concerns/with_omniauth.rb [hunk 2]
```diff
@@ -18,7 +15,7 @@ def extract_profile!(auth, user)
       user.name = auth.info.nickname || auth.info.name
       user.email = auth.info.email
       user.image_url = auth.info.image
-      user.token = auth.credentials.token
+      #user.token = auth.credentials.token
```
Comment (faloi) [inline app/models/concerns/with_omniauth.rb]: Can this be deleted?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -21,6 +22,21 @@
   <%= javascript_include_tag '//www.google.com/jsapi', 'chartkick' %>
 </head>
 <body>
+
+<script type="text/javascript">
+  var lock = new Auth0Lock('<%= ENV['MUMUKI_AUTH0_CLIENT_ID'] %>', '<%= ENV['MUMUKI_AUTH0_DOMAIN'] %>');
```
Comment (faloi) [inline app/views/layouts/application.html.erb]: You could move this env variables to an object, like `Bibliotheca::Env`

**Pull request merged**